modyn/tests/supervisor/internal/pipeline_executor/test_pipeline_executor.py

```diff
@@ -107,13 +107,15 @@ def teardown():
     shutil.rmtree(EVALUATION_DIRECTORY)
 
 
-def get_non_connecting_pipeline_executor() -> PipelineExecutor:
+def get_non_connecting_pipeline_executor(pipeline_config=None) -> PipelineExecutor:
+    if pipeline_config is None:
+        pipeline_config = get_minimal_pipeline_config()
     pipeline_executor = PipelineExecutor(
         START_TIMESTAMP,
         PIPELINE_ID,
         get_minimal_system_config(),
-        get_minimal_pipeline_config(),
-        EVALUATION_DIRECTORY,
+        pipeline_config,
+        str(EVALUATION_DIRECTORY),
```

Comment:
because this function accepts a string but `EVALUATION_DIRECTORY` is a `Path` object. Previously there is an implicit type conversion and my IDE complains about it.